docs/time.md

```diff
@@ -34,7 +34,22 @@ A representation of wall-clock time.
 
 ### time.**minus**(_duration_: string | object, _disambiguation_: 'constrain' | 'balance' | 'reject' = 'constrain') : Temporal.Time
 
-### time.**difference**(_other_: Temporal.Time) : Temporal.Duration
+### time.**difference**(_other_: Temporal.Time, _largestUnit_: string = 'days') : Temporal.Duration
+
+**Parameters:**
+- `other` (`Temporal.DateTime`): Another date/time with which to compute the difference.
+- `largestUnit` (optional string): The largest unit of time to allow in the resulting `Temporal.Duration` object.
+  Valid values are `'years'`, `'months'`, `'days'`, `'hours'`, `'minutes'`, and `'seconds'`.
+  The default is `days`.
+
+**TODO:** Rest of the documentation in another pull request, this to be rebased
+
+The `largestUnit` parameter controls how the resulting duration is expressed.
+The returned `Temporal.Duration` object will not have any nonzero fields that are larger than the unit in `largestUnit`.
+A difference of two hours will become 7200 seconds when `largestUnit` is `"seconds"`, for example.
+However, a difference of 30 seconds will still be 30 seconds even if `largestUnit` is `"hours"`.
+
+For consistency with the other types' `difference` methods, the default largest unit in the result is days, although this method never returns any duration longer than 12 hours.
```

Comment:
I don't see the benefit of this consistency, especially given that it is already broken by e.g. Temporal.Date difference not supporting "hours" or lower.

Reply:
I've changed the default value to `hours` for Time.